angular-rails-app/Gemfile

```diff
@@ -48,11 +48,12 @@ gem 'httparty', '~> 0.13.6'
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
+  gem 'sqlite3'
   gem 'byebug'
   gem 'thor', '0.19.1'
   gem 'rspec-rails', '~> 3.5'
   gem 'ffaker'
-  gem 'factory_girl_rails'
+  gem 'factory_girl_rails', :require => false
 end
```

Comment:
The Travis CI docs recommend adding "Rake" to the gem list somewhere. We're running a script instead of Rake, but still seems like good form.